mysql-test/suite/rocksdb/t/persistent_cache.test

```diff
@@ -1,41 +1,33 @@
 --source include/have_rocksdb.inc
 
---disable_warnings
-DROP TABLE IF EXISTS t1;
---enable_warnings
-
---let $_server_id= `SELECT @@server_id`
---let $_expect_file_name= $MYSQLTEST_VARDIR/tmp/mysqld.$_server_id.expect
---let $_cache_file_name= $MYSQLTEST_VARDIR/tmp/persistent_cache
---exec echo "wait" >$_expect_file_name
-
-# restart server with correct parameters
-shutdown_server 10;
---exec echo "restart:--rocksdb_persistent_cache_path=$_cache_file_name --rocksdb_persistent_cache_size_mb=100" >$_expect_file_name
---sleep 5
---enable_reconnect
---source include/wait_until_connected_again.inc
---disable_reconnect
+--let $cache_file_name=$MYSQLTEST_VARDIR/tmp/roksdb.persistent_cache
```

Comment:
Based on the comment next to the added exec rm -rf at the bottom, this is not a "file" but "directory"?